lib/lotus/cli.rb

```diff
@@ -1,110 +1,134 @@
+# TODO: Clarify/Questions
+# Create a migration along with the entity?
+# Reneame generate model => generate entity?
+# Naming slice/apps. I'd keep the slice generator, drop the app generator (slice can do everything app can)
```

Comment:
IMO, we should keep `app` and remove `slice` because `app` is more meaningful to us than `slice`.